### ISSUE TITLE
Size is calculated incorrectly

### ISSUE THREAD

**Issue by bisubus** [crisbeto/angular-svg-round-progressbar#19]
The size is calculated incorrectly, and excess padding is introduced. [Here](https://github.com/crisbeto/angular-svg-round-progressbar/blob/e300392deb07657312a4dc19ab818ea7ce434dbb/src/roundProgress.js#L44).

**Comment by crisbeto** [comment]
Hmm are you sure? When the radius is set to 100px, my circle is at exactly 200px by 200px (twice the radius). http://i.imgur.com/IDlxFbf.png

**Comment by bisubus** [comment]
I need to be more specific: the problem is the size of `<svg>` (it is [230 with stroke 15](http://i62.tinypic.com/29pypon.png) and 202 with stroke 1), not `<circle>`. Seems like it is little more complicated than just incorrect `size`, right?

**Comment by crisbeto** [comment]
Ah you're correct that the SVG element is slightly larger. I'll look into it.

**Comment by bisubus** [comment]
Great. I ended up for now with
                        element.css({
                            "width":        size + "px",
                            "height":       (isSemicircle ? size/2 : size) + "px",
                            "margin":       -stroke/2 + "px",
                            "margin-bottom": isSemicircle ? 0 : -stroke/2 + "px",
                            "overflow":     "hidden" // on some browsers the background overflows, if in semicircle mode
                        });

**Comment by crisbeto** [comment]
It was a while ago when I did this directive, but after looking at it again, I think I remember why the SVG  element's size is the diameter + 2*stroke. Its because the stroke throws off the width and height. The browser doesn't consider the stroke as a part of the width of the `circle`, in reality the circle's width and height are 230px as well. Just for fun I forced the same size on the SVG and the circle ended up overflowing. 
It probably depends on what height you expect it to be. I think it would make more sense to take off the stroke from the width and height.
